src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
```

Comment:
use Ctrl Alt L for windows for automatic formatting feature of IDEA before committing